testharness/src/main/java/bio/terra/landingzone/app/TestHarnessBeanConfig.java

```diff
@@ -9,6 +9,6 @@
 public class TestHarnessBeanConfig {
   @Bean
   public SmartInitializingSingleton postSetupInitialization(ApplicationContext applicationContext) {
-    return () -> StartupInitializer.initialize(applicationContext);
+    return () -> TestHarnessStartupInitializer.initialize(applicationContext);
```

Comment:
Needed to rename to avoid an error if both testharness and library try to upload test results:

```
Failure 1 of 1
The :testharness:jacocoTestReport task failed.
Error while creating report
> Error while analyzing /home/runner/work/terra-landing-zone-service/terra-landing-zone-service/library/build/classes/java/test/bio/terra/landingzone/app/StartupInitializer.class with JaCoCo 0.8.8.202204050719/5dcf34a.
  > Can't add different class with same name: bio/terra/landingzone/app/StartupInitializer